src/modules/launcher/Wox.Core/Plugin/PluginInstaller.cs

```diff
@@ -41,15 +41,16 @@ internal static void Install(string path)
 
                 string pluginFolderPath = Infrastructure.Constant.PluginsDirectory;
 
+                // Using InvariantCulture since this is a command line arg
                 string newPluginName = plugin.Name
-                    .Replace("/", "_")
-                    .Replace("\\", "_")
-                    .Replace(":", "_")
-                    .Replace("<", "_")
-                    .Replace(">", "_")
-                    .Replace("?", "_")
-                    .Replace("*", "_")
-                    .Replace("|", "_")
+                    .Replace("/", "_", StringComparison.InvariantCulture)
```

Comment:
StringComparison.Ordinal is a preferable alternative in this case.   Here is some documentation that discusses ordinal vs invariant culture: 

https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings

https://docs.microsoft.com/en-us/dotnet/api/system.stringcomparison?view=netcore-3.1#System_StringComparison_InvariantCulture